easybuild/easyconfigs/r/RDKit/RDKit-2024.03.5-foss-2024a.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'CMakeMake'
+
+name = 'RDKit'
+version = '2024.03.5'
+
+homepage = 'https://www.rdkit.org'
+description = "RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python."
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+source_urls = ['https://github.com/rdkit/rdkit/archive/']
+sources = ['Release_%s.tar.gz' % version.replace('.', '_')]
+checksums = ['b0870aec4dca12da5ee1100a769eaf4c71c3a6f99339f8aa72d30affc068a632']
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('Eigen', '3.4.0'),
+    ('pkgconf', '2.2.0'),
+]
+dependencies = [
+    ('Python', '3.12.3'),
+    ('SciPy-bundle', '2024.05'),
+    ('SQLite', '3.45.3'),
+    ('matplotlib', '3.9.2'),
+    ('Pillow', '11.0.0'),
+    ('Boost.Python', '1.85.0'),
+    ('Boost.Python-NumPy', '1.85.0'),
```

Comment:
is it really necessary to include both Boost.Python and Boost.Python-Numpy?

Reply:
No, I'd expect that boost.python-Numpy is the one to be kept. I'll doublecheck on Monday

Reply:
configure fails, if Boost.Python-NumPy is removed from the dependency list. Therefore I remover Boost.Python.